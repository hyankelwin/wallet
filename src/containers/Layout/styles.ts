import styled from 'styled-components';

export const Container = styled.div`
  width: 100%;
  margin: 0 auto;

  @media (max-width: 380px) {
    width: 107%;
  }

  @media (max-width: 360px) {
    width: 107%;
  }
`;
