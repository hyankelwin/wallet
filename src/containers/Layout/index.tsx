import React from 'react';

import Header from '../../components/Header';

import { Container } from './styles';

const Layout: React.FC = ({ children }) => (
  <>
    <Header />
    <Container>{children}</Container>
  </>
);

export default Layout;
