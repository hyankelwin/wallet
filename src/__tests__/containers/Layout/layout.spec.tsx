import { mount } from 'enzyme';

import Layout from '../../../containers/Layout';

describe('Layout container', () => {
  it('should be able to render Layout container with success', () => {
    const wrapper = mount(<Layout />);

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
