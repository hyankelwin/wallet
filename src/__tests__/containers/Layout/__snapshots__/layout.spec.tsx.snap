// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout container should be able to render Layout container with success 1`] = `
"<Layout>
  <Header>
    <styled.div>
      <div className=\\"sc-bdvvtL gfWDWR\\">
        <div>
          <HeaderImage>
            <styled.div>
              <div className=\\"sc-gsDKAQ gwkgJf\\" />
            </styled.div>
          </HeaderImage>
        </div>
        <div>
          <p>
            Your wallet
          </p>
          <p>
            digital
          </p>
        </div>
      </div>
    </styled.div>
  </Header>
  <styled.div>
    <div className=\\"sc-dkPtRN iknFHz\\" />
  </styled.div>
</Layout>"
`;
